defaults/main.yml

```diff
@@ -30,6 +30,10 @@ php_fpm_pm_start_servers: 5
 php_fpm_pm_min_spare_servers: 5
 php_fpm_pm_max_spare_servers: 5
 
+php_fpm_access_log: /var/log/php-fpm/access.log
```

Comment:
All the options would need to be documented in the README as well.

Also, for `php_fpm_access_format`, is that the default access format?

Reply:
``access.log`` by default is not defined. How we can handle that?

Reply:
That's a good question...

I think at some point, it's easier to just template the php-fpm configuration from a .j2 file in your playbook instead of trying to define things for any of the possible options :/

But access log would be nice to be able to configure. I just like to stick to PHP's defaults if possible.

Reply:
I have modified php_fpm_access_format with the default value.
But what's not default is having a log file